doc/stdlib/math.rst

```diff
@@ -1684,6 +1684,12 @@ Statistics
 
    Compute the mean of whole array ``v``\ , or optionally along the dimensions in ``region``\ . Note: Julia does not ignore ``NaN`` values in the computation. For applications requiring the handling of missing data, the ``DataArray`` package is recommended.
 
+.. function:: mean(f::Function, v)
+
+   .. Docstring generated from Julia source
+
+   Apply the function `f` to each element of `v` and take the mean.
```

Comment:
Uh, yes... Is that not what I was supposed to do? I used the other entry for `mean` in that document as a template for this one. `mean` doesn't actually have a docstring in the source anywhere that I could find.